packages/blockchain-wallet-v4-frontend/src/scenes/Reset2FAToken/index.js

```diff
@@ -20,7 +20,7 @@ const Wrapper = styled.div`
 class Reset2FAToken extends React.PureComponent {
   constructor (props) {
     super(props)
-    this.state = { token: decodeURIComponent(props.location.pathname.split('/reset2fa-token/')[1]) }
+    this.state = { token: decodeURIComponent(props.location.pathname.split('/reset-two-factor/')[1]) }
```

Comment:
Its a bit confusing, there are 2 reset two factor auth links. One is in the wallet (reset-2fa) and starts the reset process, the second (above, reset-two-factor) is an external link from an email that the wallet parses a token from.